lib/utils.js

```diff
@@ -67,6 +76,12 @@ function normalizeType(params) {
       override.match = new RegExp(overrideDef.match, 'i');
     }
 
+    // If the override doesn't provide elevated_limits use the ones defined in the upper level (if any)
+    if (!override.elevated_limits && type.elevated_limits) {
+      override.elevated_limits = type.elevated_limits;
+      override.erl_activation_period_seconds = type.erl_activation_period_seconds;
```

Comment:
Isn't `erl_activation_period_seconds` be a part of `type.elevated_limits` (or `override.elevated_limits`) already? It's not expected to be at the `override` level.

Reply:
Seems to be still there :D Forgot to push maybe?

Reply:
ups forgot to add it to the commit